ludwig/features/base_feature.py

```diff
@@ -164,7 +164,8 @@ def call(
             self,
             inputs,  # hidden, other_output_hidden
             training=None,
-            mask=None
+            mask=None,
+            targets=None
```

Comment:
I'm not sure this is allowed.
`call()` implements the abstract method from the superclass and my understading is that that method has only 3 parameters: `inputs`, `training`, and `mask`. So this breaks the method signature. But i may be wrong and adding additional arguments may be alowed in python, i have to doublecheck.
But if I am correct, the alternative is to pass `(inputs, targets)` as inputs instead and having an if that checks if it's a inputs is a tuple or not.

Reply:
Change signature to comply with `inputs`, `training` and `mask`.  Commit https://github.com/uber/ludwig/pull/699/commits/3310a13ab89c4abebdb3c6be753368ae165a68cb